### PR TITLE
write  linq

### PR DESCRIPTION
**Pull request by JillyMan**

### CODE REVIEW

File: Module-6/Task/Task/LinqSamples.cs [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+				ObjectDumper.Write(p);
+			}
+		}
+
+		[Category("Restriction Operators")]
+		[Title("Where - Task 1")]
+		[Description("Выдайте список всех клиентов, чей суммарный оборот (сумма всех заказов) превосходит некоторую величину X. Продемонстрируйте выполнение запроса с различными X")]
+		public void Linq1()
+		{
+			var total = 40020;
```
Comment (GeorgeManiya) [inline Module-6/Task/Task/LinqSamples.cs:69]: Please show how this query is working with different total numbers as it is expected in the task description.

File: Module-6/Task/Task/LinqSamples.cs [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+		[Description("Для каждого клиента составьте список поставщиков, находящихся в той же стране и том же городе. Сделайте задания с использованием группировки и без")]
+		public void Linq2()
+		{
+			var query = dataSource.Customers
+					.Select(x => new
+					{
+						Customer = x,
+						Suppliers = dataSource.Suppliers.Where(s => s.City == x.City && s.Country == x.Country)
+					});
+			//TODO: MAKE WITH GROUP
```
Comment (GeorgeManiya) [inline Module-6/Task/Task/LinqSamples.cs:85]: Please add solution with grouping method.

File: Module-6/Task/Task/LinqSamples.cs (hunk 1 above)
Comment (JillyMan) [reply Module-6/Task/Task/LinqSamples.cs:69]: added

File: Module-6/Task/Task/LinqSamples.cs (hunk 2 above)
Comment (JillyMan) [reply Module-6/Task/Task/LinqSamples.cs]: solution was added

Comment (GeorgeManiya) [reply Module-6/Task/Task/LinqSamples.cs]: lgtm

File: Module-6/Task/Task/LinqSamples.cs (hunk 1 above)
Comment (GeorgeManiya) [reply Module-6/Task/Task/LinqSamples.cs:69]: you need to show how exact same query can work with different values. E.g. `var total = x;` - run query - `total = y` - run query again

Comment (JillyMan) [reply Module-6/Task/Task/LinqSamples.cs]: need review again

**Pull request merged**